app/models/wpcc/period.rb

```diff
@@ -14,8 +14,10 @@ def highest_employee_percent
       Percent.choose_highest(@employee_percent, @user_input_employee_percent)
     end
 
-    def highest_employer_percent
-      Percent.choose_highest(@employer_percent, @user_input_employer_percent)
+    def required_employer_percent(salary)
+      return @user_input_employer_percent if salary_below_minimum?(salary)
+
+      highest_employer_percent
```

Comment:
Can we change the naming to be eligible_salary in all "salary" variables names for this class? It is just because when I was reading the code I was in doubt if was the salary from the step 1 OR the eligible salary (which on this class, is eligible salary). Or qualified earnings, I don't know what name is being referred to but just "salary" can confuse when is read. Wdyt?